projections/account_test.go

```diff
@@ -18,23 +18,24 @@ func Test_AccountProjectionHandler(t *testing.T) {
 			database, db := openDB(context.Background())
 			defer database.Close()
 
-			testkit.New(&example.App{ReadDB: db}).
-				Begin(
-					t,
-					// TODO: Isolate test by handler.
-					// See https://github.com/dogmatiq/testkit/issues/56
-					testkit.WithOperationOptions(
-						engine.EnableProcesses(false),
-						engine.EnableProjections(true),
-					),
-				).
-				Prepare(
+			Begin(
+				t,
+				&example.App{ReadDB: db},
+				// TODO: Isolate test by handler.
+				// See https://github.com/dogmatiq/testkit/issues/56
+				WithUnsafeOperationOptions(
```

Comment:
Instead of `WithUnsafeOperationOptions()` you can use the `test.EnableHandlers()` method.